client/src/components/Event.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { EventProps } from "../types";
+import EventDatesTable from "./EventDatesTable";
+
+const Event: React.FC<EventProps> = ({ location = "unknown", id, title, dates }) => {
+  return (
+    <div id={id}>
+      <h2>{`Event: ${title}`}</h2>
+      <h3>{`Location: ${location}`}</h3>
```

Comment:
Mozna bych zauvazoval ten nadpis h3 vubec neukazovat, pokud location neni (a tim bych i odstranil tu default value "unknown" vyse). Mohlo by to byt lepsi nez ukazat "unknown". Ale zavisi, na konkretnim pripade resp. projetku, co je za pozadavky